Makefile

```diff
@@ -487,3 +492,22 @@ define list-running-pods
 	@echo "\n$(YELLOW)Local cluster $(CYAN)$1 $(NC)"
 	kubectl get pods -A --context=k3d-$1
 endef
+
+define deploy-prometheus
+	@echo "\n$(YELLOW)Local cluster $(CYAN)$1$(NC)"
+	kubectl config use-context k3d-$1
+
+	@echo "\n$(YELLOW)Set annotations on pods that will be scraped by prometheus$(NC)"
+	kubectl annotate pods -l name=k8gb -n k8gb --overwrite prometheus.io/scrape="true"
+	kubectl annotate pods -l name=k8gb -n k8gb --overwrite prometheus.io/path=/metrics
```

Comment:
prometheus.io/path: If the metrics path is not /metrics, define it with this annotation.

Reply:
@k0da , I was following recommended way: https://artifacthub.io/packages/helm/prometheus-community/prometheus#scraping-pod-metrics-via-annotations

Reply:
This is the same as default, it not worth to annotate it explicitly

Reply:
@kuritka , there's no need to install full-fledged Prometheus if you want to check or test the output of the metrics endpoint. All you have to do is to set up port-forwarding to the k8gb controller pod with `kubectl`
```
kubectl port-forward -n k8gb pod/k8gb-XXXXXXXX-YYYYY 8080:8080
curl localhost:8080/metrics
```
(You can also temporarily expose the endpoint using Service)

Reply:
@k0da , amended

Reply:
@sommaritane, thanks. `make deploy-prometheus` and then have an accessible web UI without any further steps was the intention. I will definitely test the automatic way. Still, it is better (and not only for the scope implementation of this task) that the k8gb developer has the possibility to quickly check the metrics in prometheus.